cnxepub/formatters.py

```diff
@@ -775,9 +775,13 @@ def _replace_exercises(elem):
 def html_listify(tree, root_xl_element, extensions, list_type='ol'):
```

Comment:
Yeah I had to dig for that, too - it's the extension to use for the filename for this object. It's "xhtml" for almost everything in our epubs. I'm using this as a handy way to detect the objects that don't have a file: i.e. subcollections. I'll add some comments